packages/terra-application-header-layout/src/terra-dev-site/doc/example/HeaderWireframe.jsx

```diff
@@ -48,10 +48,10 @@ const HeaderWireframe = () => {
     <div className={cx('content-wrapper')}>
       <div className={cx('responsive-element-wrapper')}>
         <ResponsiveElement
-          defaultElement={tinyHeader}
           tiny={tinyHeader}
-          small={smallHeader}
-          medium={mediumHeader}
+          small={tinyHeader}
```

Comment:
We should be able to delete the `small` prop here, it will fallback to what's defined for `tiny`